src-electron/generator/helper-zap.js

```diff
@@ -465,6 +465,15 @@ function multiply() {
   return nums.reduce((prev, next) => prev * next, 1)
 }
 
+/**
+ * Returns a boolean based on whether a given value is a power or 2 or not.
+ * @param {*} val
+ * @returns boolean
+ */
+function is_power_of_two(val) {
```

Comment:
Seems completerly unrelated to the base type stuff.

Reply:
Yeah it is not needed but is useful for determining odd sized data types in the future.